assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -165,472 +328,91 @@ header[role="banner"] {
-    background-color: gu-colour(news-garnett-highlight);
-  }
-
-  // TODO: center with flex-box instead!
-  .glogo {
-    top: 7px;
-    right: 10px;
-  }
-
-  .gu-content__main {
-    display: flex;
-    flex-direction: column-reverse;
-  }
-
-  .gu-content__bg {
-    display: none;
-  }
-
-  .gu-content__bg .svg-contributions-bg-desktop {
-    display: none;
-  }
-
-  .gu-content__bg .svg-contributions-bg-mobile {
-    bottom: 0;
-    height: 223px;
-    left: 50%;
-    position: absolute;
-    transform: translate(-50%,0);
-    width: 100%;
-  }
-
-  .countryGroups {
-    margin-left: 10px;
-    text-overflow: ellipsis;
-    width: 185px;
-    white-space: nowrap;
-  }
-
-  .countryGroups summary {
-    padding-left: 10px;
-  }
-
-  .countryGroups .svg-globe {
-    display: none;
-  }
-
-  .countryGroups__item > a {
-    padding-left: 10px;
-  }
-}
-
-@include mq($from: tablet) {
-  header[role="banner"] {
-    padding: 0 calc(50% - 350px);
-    width: 700px;
-  }
-
-  footer[role="contentinfo"] {
-    padding: 0 calc(50% - 350px) ($gu-v-spacing * 2);
-    width: 700px;
-  }
-
-  .glogo {
-    top: 7px;
-  }
-
-  .gu-content__content {
-    border-color: gu-colour(garnett-neutral-4);
-    border-style: none solid;
-    border-width: 0 1px;
-    width: 380px;
-    margin-left: calc(50% - 361px); /* account for the 11px of negative margin */
-  }
-
-  .gu-content__bg {
-    height: 580px;
-    left: 0;
-    position: absolute;
-    top: 245px;
-    width: 100%;
-    z-index: -1;
-  }
-
-  .gu-content__bg svg {
-    bottom: 0;
-    height: 580px;
-    left: 50%;
-    position: fixed;
-    transform: translate(-50%,0);
-    width: 1400px;
-  }
-
-  .gu-content__main .svg-contributions-bg-mobile {
-    display: none;
-  }
-}
-
-@include mq($from: leftCol) {
-  header[role="banner"] {
-    padding: 0 calc(50% - 550px) 0 calc(50% - 390px);
-    width: 940px;
-  }
-
-  footer[role="contentinfo"] {
-    padding: 0 calc(50% - 550px) ($gu-v-spacing * 2) calc(50% - 390px);
-    width: 940px;
-  }
-
-  .gu-content__content {
-    width: 380px;
-    margin-left: calc(50% - 401px); /* account for the 11px of negative margin */
-  }
-}
-
-@include mq($from: wide) {
-  header[role="banner"] {
-    padding: 0 calc(50% - 620px);
-    width: 1240px;
-  }
-
-  footer[role="contentinfo"] {
-    padding: 0 calc(50% - 620px) ($gu-v-spacing * 2);
-    width: 1240px;
-  }
+.countryGroups__item > a:hover,
+.countryGroups__item > a:focus {
+  background: gu-colour(garnett-neutral-3);
+  font-weight: bold;
 }
 
-@supports(display: grid) {
-  .gu-content {
-    display: grid;
-    grid-template-rows: [header-t] 75px [header-b] 1fr [footer-t] auto [footer-b];
-    grid-template-columns: 1fr;
-  }
-
-  header[role="banner"] {
-    height: auto;
-    grid-row: header-t / header-b;
-  }
 
-  footer[role="contentinfo"] {
-    grid-row: footer-t / footer-b;
-  }
+/*= FOOTER */
 
-  .gu-content__main {
-    grid-row: header-b / footer-t;
-  }
+footer[role="contentinfo"] {
```

Comment:
maybe we could get rid of these tag/attribute selectors while we're at it, and replace with classes